kuma/wiki/jinja2/wiki/document.html

```diff
@@ -356,6 +356,14 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                   {% endif %}
                 </div>
               {% endif %}
+
+              {% if waffle.switch('newsletter_article') %}
```

Comment:
The include has the newsletter switch in it. 

Hm. If this is confusing maybe I should find another way to achieve the same thing.